config/session.js

```diff
@@ -10,44 +10,49 @@
  */
 
 module.exports.session = {
-  /***************************************************************************
-   *                                                                          *
-   * Session secret is automatically generated when your new app is created   *
-   * Replace at your own risk in production-- you will invalidate the cookies *
-   * of your users, forcing them to log in again.                             *
-   *                                                                          *
-   ***************************************************************************/
-  secret: "cf1e0dd462546522d7cebec3f4b841fc",
+   /***************************************************************************
+    *                                                                          *
+    * Session secret is automatically generated when your new app is created   *
+    * Replace at your own risk in production-- you will invalidate the cookies *
+    * of your users, forcing them to log in again.                             *
+    *                                                                          *
+    ***************************************************************************/
+   secret: "cf1e0dd462546522d7cebec3f4b841fc",
```

Comment:
This seems like something we don't actually want to check in to our code base.  

Should we put it in an environment variable? or some other form?

Reply:
The installer will now generate a unique session id in `local.js` (https://github.com/digi-serve/ab-cli/pull/49). It will supersede this secret.